core/src/test/java/org/apache/accumulo/core/util/NumUtilTest.java

```diff
@@ -22,12 +22,15 @@
 import static org.apache.accumulo.core.util.NumUtil.bigNumberForSize;
 import static org.junit.Assert.assertEquals;
 
+import java.util.Locale;
+
 import org.junit.Test;
 
 public class NumUtilTest {
 
   @Test
   public void testBigNumberForSize() {
+    Locale.setDefault(Locale.US);
```

Comment:
This change seems unrelated. Is it necessary? If it's an issue, it could be a separate PR.

Reply:
I created https://github.com/apache/accumulo/pull/2072 - that will separate this issue into its own commit. Once that change is merged, this can be re-based and merged.